recipes/tinyobjloader/2.x/conanfile.py

```diff
@@ -0,0 +1,51 @@
+import os
```

Comment:
I see no difference between 1.x and 2.x maybe we can re-use the same recipe for both.

Reply:
The only difference between 1.x and 2.x is the API, so the test case is different.
Is there a way to deduplicate the rest of the code?

Reply:
I need to check if we can customize the test_package name. Let me see it.

Reply:
unfortunately we don't have such feature, but we can re-use the same test_package folder for both tests. Using cmake features we can detect the package reference and build the correct test. Lemme check some good approach.